site/guide/guides.qmd

```diff
@@ -153,6 +153,8 @@ Set up validation guidelines and prepare validation reports, work with findings
 :::{#guides-model-validation}
 :::
 
```

Comment:
```suggestion
Watch a full walkthrough on model validation: 
```

Reply:
@nrichers Can I ask why you rejected this change? For me, users don't usually want to click on videos unless they're clear on what value they bring.

Reply:
@validbeck two things, the original change breaks the video alignment but after looking at it in the preview, I don't think the text actually adds anything. I love the simple look of the listing tiles — the video just underneath those without any additional text looks very clean. Add the next, not so much.

<img width="965" alt="image" src="https://github.com/user-attachments/assets/9a94551d-2596-4f43-902c-a57672b4feb6">

Reply:
Yes, ideally there should be a line break and THEN the video — apologies for not adding that in the suggestion. Can we try that and see how it looks?

Reply:
You can but today's storm in a teacup is all about whether you need lead-in text at all: https://github.com/validmind/documentation/pull/450#issuecomment-2377559283

('Add lightness' would suggest you practice radical simplicity and save the words. Our docs site is already word heavy.)